src/chocolatey/infrastructure.app/services/PythonService.cs

```diff
@@ -272,7 +272,7 @@ public string build_args(ChocolateyConfiguration config, IDictionary<string, Ext
 
             args = args.Replace(LOG_LEVEL_TOKEN, config.Debug ? "-vvv" : "");
 
-            if (config.CommandName.is_equal_to("intall"))
+            if (config.CommandName.is_equal_to("install"))

```

Comment:
?

Reply:
Oh wow!  This is a lesser used portion of the Chocolatey application, but I would have still expected for this to have been caught before now.  This change makes sense to me.